doc/ug/dv_methodology.md

```diff
@@ -373,22 +373,22 @@ The following are some of the best practices when adding exclusions:
 
    These categories are as follows:
 
-  *  **UNR**: Unreachable code due to constraints, or module inputs being tied off in a certain way will result in specific coverage items to be unreachable.
-     Additional explanation is optional.
-  *  **NON_RTL**: Simulation constructs in RTL that can be safely excluded in structural coverage collection.
-     These include tasks and functions, initial / final blocks that are specifically used for simulation such as backdoor write and read functions for memory elements.
-     Additional explanation is optional.
-  *  **UNSUPPORTED**: Item being excluded is a part of design feature that is not supported.
-     Additional explanation is optional.
-     *  IP designed by some other team / third party is incorporated, but only a subset of the features are in use. Remaining ones are not supported.
-     *  Features that are added into the design but are not made a part of the design specification for the current generation / chip being taped out
-     *  UVC / agent with detailed coverage items where certain crosses are not supported by the design (ex: TL agent with fcov on full spectrum of burst with all sizes and lengths, but only a subset of it actually being supported).
-     Additional explanation is mandatory.
-  *  **EXTERNAL**: Items that are already covered in another bench.
-     Additional explanation is mandatory.
-  *  **LOW_RISK**: Items that are prohibitively hard to hit, given the resource constraints and are deemed to be of low risk and low value.
-     Features that are added to the design AND are described adequately in the design spec AND a collective upfront decision has been made in agreement with SW/architecture/design/DV to not verify it due to resource constraints.
-     Additional explanation is mandatory.
+   *  **UNR**: Unreachable code due to constraints, or module inputs being tied off in a certain way will result in specific coverage items to be unreachable.
```

Comment:
`items to be unreachable` -> `items being unreachable`

Reply:
Fixed.